src/localizers/oldGlobalize.js

```diff
@@ -36,6 +36,7 @@ export let formats = {
   agendaHeaderFormat: dateRangeFormat,
 
   agendaDateFormat: 'ddd MMM dd',
+  agendaSingleTimeFormat: 't',
```

Comment:
sorry, same for this one, I think `agendaTimeFormat` should be good here without the need for a new format